MonMaraicher/Features/FarmerDetail/FarmerDetailsView.swift

```diff
@@ -99,39 +88,14 @@ private extension FarmerDetailsView {
             Marker(viewModel.title, systemImage: viewModel.markerSystemImageName, coordinate: viewModel.coordinate)
                 .tint(.orange)
         }
-        .aspectRatio(1.5, contentMode: .fit)
+        .aspectRatio(2, contentMode: .fit)
         .clipShape(.rect(cornerRadius: 16))
-        .shadow(radius: 4)
+        .shadow(radius: 8)
         .allowsHitTesting(false)
         .mapControlVisibility(.hidden)
     }
 
-    private var directionButton: some View {
-        Button {
-            viewModel.onItineraryButtonTapped()
-        } label: {
-            Text(viewModel.directionButtonTitle)
-        }
-        .font(.headline)
-    }
-
-    private var phoneButton: some View {
-        VStack {
-            if viewModel.phoneNumber == nil {
-                Image(systemName: "phone.fill")
-            } else {
-                if let number = viewModel.phoneNumber {
-                    if let url = URL(string: "tel:\(number)") {
-                        Link(destination: url) {
-                            Image(systemName: "phone.fill")
-                        }
-                    }
-                }
-            }
-        }
-    }
-
-    private var addressSection: some View {
+    private var informationsSection: some View {
```

Comment:
Ok but informations might be too generic. Everything on screen is actually an information ;)